niftypad/api/__init__.py

```diff
@@ -7,15 +7,23 @@
 log = logging.getLogger(__name__)
 
 
-def kinetic_model(src, dst=None, params=None, model='srtmb_basis', input_interp_method='linear',
-                  w=None, r1=1, k2p=0.000250, beta_lim=None, n_beta=40, linear_phase_start=500,
-                  linear_phase_end=None, km_outputs=None, thr=0.1, fig=False):
+# def kinetic_model(src, dst=None, params=None, model='srtmb_basis', input_interp_method='linear',
+#                   w=None, r1=1, k2p=0.000250, beta_lim=None, n_beta=40, linear_phase_start=500,
+#                   linear_phase_end=None, km_outputs=None, thr=0.1, fig=False):
+
+def kinetic_models(src, dst=None, params=None, models=['srtmb_basis','logan_ref'], input_interp_method='linear',
```

Comment:
not sure about this @JJiao...

- why try to do multiple models in one function? Why not just call the function multiple times (once per model)?
- `linear_phase_(start|end)_(l|l2|m|m2)` seems very clunky (again, old API seems much cleaner)

Reply:
maybe better to do this:

```python
def kinetic_models(models=[...], **kwargs):
    results = []
    for model in models:
        results.append(kinetic_model(**kwargs))
    return results
```

Reply:
Sure! Let's use 'kinetic_modelS'